development/build/scripts.js

```diff
@@ -612,6 +613,8 @@ function getEnvironmentVariables({ buildType, devMode, testing }) {
         ? process.env.SEGMENT_PROD_LEGACY_WRITE_KEY
         : metamaskrc.SEGMENT_LEGACY_WRITE_KEY,
     SWAPS_USE_DEV_APIS: process.env.SWAPS_USE_DEV_APIS === '1',
+    ONBOARDING_V2:
+      process.env.ONBOARDING_V2 === '1' || metamaskrc.ONBOARDING_V2 === '1',
```

Comment:
This should be equivalent:
```suggestion
    ONBOARDING_V2: metamaskrc.ONBOARDING_V2 === '1',
```

Since `process.env.ONBOARDING_V2` is set as the default on line 30.